scripts/install.js

```diff
@@ -37,7 +37,7 @@ var ninja_vendor_dir = path.join(root_dir, 'vendor', 'ninja-build')
 
 function build_ninja() {
     console.log('No prebuilt Ninja, building Ninja now')
-    var build_ninja_command = "tar -xf  ninja-1.7.2.tar.gz  && cd  ninja-1.7.2  && ./configure.py --bootstrap "
+    var build_ninja_command = "tar -xzf  ninja-1.7.2.tar.gz  && cd  ninja-1.7.2  && ./configure.py --bootstrap "
```

Comment:
The documentation also says that the option is ignored in extract mode - it would be great if you can test that, thanks @bassjacob. I can add a conditional to handle that case if needed.